Documentation/user-guides/running-exporters.md

```diff
@@ -68,7 +68,7 @@ spec:
     interval: 15s
 ```
 
-A better example for monitoring Kubernetes cluster components can be found in [Cluster Monitoring](cluster-monitoring.md).
+A better example for monitoring Kubernetes cluster components can be found in [here](monitoring-kubernetes-ingress.md)
```

Comment:
Maybe?

Reply:
again @brancz ^ :)

Reply:
same, this should link to kube-prometheus